components/inquiry_questionnaire/views/specialist.coffee

```diff
@@ -35,7 +35,7 @@ module.exports = class Specialist extends StepView
     form.state 'loading'
 
     @__serialize__ = Q.all [
-      @inquiry.save _.extend { contact_gallery: false }, form.data()
+      @inquiry.save _.extend form.data(), { contact_gallery: false }
```

Comment:
Since the default added in the hidden field to the form is `contact_gallery: false`, we have to overwrite it here for specialist. I'd like to point out that the pattern here is different than in the [inquiry.coffee](https://github.com/artsy/force/pull/386/files#diff-520520eece795934fabe20cb7e9af448R53).